cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/elements/ControlAreaEq.java

```diff
@@ -30,7 +30,7 @@ public static void write(String id, String controlAreaName, String energyIdentif
         writer.writeCharacters(energyIdentificationCodeEIC);
         writer.writeEndElement();
         writer.writeEmptyElement(cimNamespace, "ControlArea.type");
-        writer.writeAttribute(RDF_NAMESPACE, CgmesNames.RESOURCE, CONTROL_AREA_TYPE);
+        writer.writeAttribute(RDF_NAMESPACE, CgmesNames.RESOURCE, cimNamespace + CONTROL_AREA_TYPE);
```

Comment:
Isn't there a '#' missing here?

Reply:
All namespaces finish with a dash character, `#`, so I think it is ok.